app-tasks/rf/src/rf/uploads/landsat8/create_scenes.py

```diff
@@ -0,0 +1,79 @@
+"""Creates scenes for landsat 8 imagery given a csv row"""
+
+
+import logging
+
+
+from rf.models import Scene
+from rf.utils.io import JobStatus, Visibility
+
+from .create_thumbnails import create_thumbnails
+from .create_footprint import create_footprint
+from .settings import organization
+from .io import get_landsat_path
+
+
+logger = logging.getLogger(__name__)
+
+
+def filter_empty_keys(kv_dict):
+    """Returns the dictionary that is the subset of kv_dict with empty values removed.
+    This method will not recurse into the dictionary to remove empty values from any
+    nested dictionaries, so performance is only guaranteed for one-level deep dicts.
+
+    Args:
+        kv_dict (dict): one level deep dictionary
+
+    Returns:
+        dict
+    """
+
+    ok_values = [0, False]
+
+    return {k: v for k, v in kv_dict.iteritems() if v or v in ok_values}
+
+
+def create_landsat8_scenes(csv_row):
+    """Returns scenes that can be created via API given a path to tiles for Landsat 8
+
+    Args:
+        csv_row (dict): value returned by a call to DictReader.next on the tiles csv
+
+    Returns:
+        List[Scene]
+    """
+
+    scene_id = csv_row.pop('sceneID')
+    timestamp = csv_row.pop('acquisitionDate') + ' 00:00:00'
+    cloud_cover = float(csv_row.pop('cloudCoverFull'))
+    sun_elevation = float(csv_row.pop('sunElevation'))
+    sun_azimuth = float(csv_row.pop('sunAzimuth'))
+    landsat_path = get_landsat_path(scene_id)
+
+    datasource = 'Landsat 8'
+    tags = ['Landsat 8', 'JPEG2000']  # TODO I think JPEG2000 is right because QGIS picked up WGS84 immediately?
+
+    scene_metadata = filter_empty_keys(csv_row)
+
+    scene = Scene(
+        organization,
+        0,
+        Visibility.PUBLIC,
+        30,
+        tags,
+        datasource,
+        scene_metadata,
+        'L8 {} {}'.format(landsat_path, '30m'),  # name
```

Comment:
I think we're going to need to create two "scenes" for Landsat 8.

 1. 15m scene that has band 8
 2. 30m scene that will have the rest of the bands

See http://landsat.usgs.gov/band_designations_landsat_satellites.php